examples/cookbook/effects/nested_nav/analysis_options.yaml

```diff
@@ -1,6 +1,5 @@
-include: package:flutter_lints/flutter.yaml
+# Take our settings from the example directory's main analysis_options.yaml
+# file. If necessary for a particular example, this file can also include
+# overrides for individual lints.
 
-linter:
-  rules:
-    avoid_print: true
-    prefer_single_quotes: true
+include: ../../../analysis_options.yaml
```

Comment:
Looks like all of the analysis_options.yaml imports are missing newline at the end of the file

Reply:
And there is only a hundred of so of 'em =)

Reply:
On the upside, only 31 or so had trailing newlines.